README.md

```diff
@@ -77,7 +77,9 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 ## 📋 Requirements
 
-- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
+- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). Note Linux
+ is only partially supported (serving and testing a trained model does not work on Linux).
```

Comment:
Is this accurate? If so, can we get the block lifted @leseb @chudegao 🙏 

```suggestion
- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). Note Linux
 is not fully supported (testing a trained model does not currently work on Linux).
```

Reply:
This will need to be added to the PR before approval.